### PR TITLE
Backport fix geocoded proposals

### PR DESCRIPTION
**Pull request by Quentinchampenois**
### Description

Fix bug when latitude or longitude is kind of "NaN" and break the map.

#### To reproduce

* Log in as admin
* Go to admin process
* Enable geocoding on proposals
then
* Create a new proposal with valid address with title "Proposal should be fixed now"
map should work

Then try to reproduce bug using Rails console
* In a new terminal window open rails console `bundle exec rails console`
* In console type `Decidim::Proposals::Proposal.find_by(title: {"en"=>"Proposal should be fixed now"}).update(latitude: 0.0 / 0.0)`
* Then navigate to proposals index page and map should be still displayed

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -265,7 +265,7 @@
   end
 
   create_table "decidim_awesome_config", force: :cascade do |t|
-    t.jsonb "var"
+    t.string "var"
```
Comment (Quentinchampenois) [inline db/schema.rb:268]: Here I can't test if it breaks the awesome config on the current instance

Comment (moustachu) [reply db/schema.rb:268]: @Quentinchampenois 
Where does that change come from ? 
It's strange to have a change in the schema.rb (meaning there was a db:schema:dump) **without** a new migration.

Comment (armandfardeau) [reply db/schema.rb:268]: I suggest that you reverse this file to keep it outside of the P.R.

**Review by armandfardeau — CHANGES_REQUESTED** [review]

**Comment by Quentinchampenois** [comment]
Thanks for the review, I've updated the PR
@moustachu @armandfardeau

**Review by paulinebessoles — APPROVED** [review]

**Review by moustachu — APPROVED** [review]

**Review by armandfardeau — APPROVED** [review]

**Pull request merged**